ci-operator/step-registry/rosa/idp/htpasswd/rosa-idp-htpasswd-commands.sh

```diff
@@ -44,7 +44,7 @@ IDP_PASSWD=$(openssl rand -base64 15)
 rosa create idp -c ${CLUSTER_ID} \
                 -y \
                 --type htpasswd \
-                --name htpasswd-1 \
+                --name rosa-htpasswd \
```

Comment:
I am running the steps, let me do this update in my PR https://github.com/openshift/release/pull/30943

Reply:
Done. The change has been take out from this PR.